test/src/waittask.spec.ts

```diff
@@ -29,122 +28,6 @@ describe('waittask specs', function () {
   setupTestBrowserHooks();
   setupTestPageAndContextHooks();
 
-  describe('Page.waitFor', function () {
```

Comment:
I think some of the tests don't use the deprecated waitFor? I think we should keep them (I wonder if the coverage test would complain)

Reply:
It's okay because we tests these functions in other places.